packages/components/src/toggle-group-control/toggle-group-control/component.tsx

```diff
@@ -83,12 +84,11 @@ function ToggleGroupControl(
 	const classes = useMemo(
 		() =>
 			cx(
-				styles.ToggleGroupControl,
+				styles.ToggleGroupControl( { size } ),
 				isBlock && styles.block,
```

Comment:
I kind of wanted to refactor this so these styles go in the `styles.ts` file, but there were some complications. Better done in a separate PR.